frame/contracts/src/tests.rs

```diff
@@ -2899,3 +2899,29 @@ fn contract_reverted() {
 		assert_eq!(result.data.0, buffer);
 	});
 }
+
+#[test]
+fn code_rejected_error_works() {
```

Comment:
Is there any way to test that the debug buffer was written to correctly?

Reply:
Yes. Added the check.